google-cloud-tasks/lib/google/cloud/tasks/v2beta2/cloudtasks_services_pb.rb

```diff
@@ -49,7 +49,7 @@ class Service
             # WARNING: Using this method may have unintended side effects if you are
             # using an App Engine `queue.yaml` or `queue.xml` file to manage your queues.
             # Read
-            # [Overview of Queue Management and queue.yaml](/cloud-tasks/docs/queue-yaml)
+            # [Overview of Queue Management and queue.yaml](/tasks/docs/queue-yaml)
```

Comment:
Where is this file? Is it supposed to be added with the generated files?

Reply:
These are coming from the documentation in the proto itself. It's wrong; I believe the intent is for the link to be a cloud.google.com page, but it shouldn't be making that assumption. I'll open an internal issue to fix it. For now, it is what it is. :(

Reply:
For the record, internal issue b/111358188 has been opened to fix the documentation string in the protos.